db/seeds.d/07-provisioning_templates.rb

```diff
@@ -79,6 +80,13 @@
       :snippet  => false,
       :template => File.read(File.join("#{Rails.root}/app/views/unattended", input.delete(:source)))
     }.merge(input))
+
+    created.push(t.name)
```

Comment:
why not just updating the template taxonomies here as opposed to searching for them by name later?